packages/website/src/components/inputs/CopyButton.module.css

```diff
@@ -0,0 +1,62 @@
+.copyButton {
+  --ifm-button-padding-horizontal: 1rem;
+  --ifm-button-background-color: var(--ifm-background-surface-color);
+  --ifm-button-color: var(--ifm-color-emphasis-700);
+  --ifm-button-border-color: var(--ifm-color-emphasis-200);
+  --copy-button-icon-check: 0;
+  --copy-button-icon-copy: 1;
+
+  font-size: 0.75rem;
+  position: absolute;
+  top: 0;
+  right: 0;
+  align-items: center;
+  display: flex;
+  line-height: 0;
+  padding: 0.4rem;
+  transition: opacity 0.2s ease-in-out, border-color 0.4s ease-in-out,
+    color 0.4s ease-in-out;
+}
+
+.copyButton:disabled {
+  --ifm-button-background-color: var(--playground-secondary-color);
+  --ifm-button-color: var(--ifm-color-success-dark);
+  --ifm-button-border-color: var(--ifm-color-emphasis-200);
+  --copy-button-icon-check: 1;
+  --copy-button-icon-copy: 0;
+
+  opacity: 1;
+}
+
+.copyButton:hover,
+.copyButton:focus-visible {
+  --ifm-button-background-color: var(--playground-secondary-color);
+  --ifm-button-border-color: var(--ifm-color-emphasis-300);
+}
+
+.copyIcon {
+  opacity: var(--copy-button-icon-copy);
+  position: absolute;
+  transition: opacity 0.2s ease-in-out;
+}
+
+.checkIcon {
+  opacity: var(--copy-button-icon-check);
+  transition: opacity 0.2s ease-in-out;
+}
+
+.copyButtonTooltip::before {
+  margin-right: 7px;
+}
+
+.copyButtonTooltip::after {
+  min-height: 25px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  right: 23px !important;
```

Comment:
This doesn't seem to do anything on my machine? Also, avoiding `!important` would be nice.

```suggestion
```

Btw, at this point I feel like maybe it'd make sense for us to go with a UI library for the tooltip & such, instead of rolling it outselves? There are accessibility considerations that I don't fully understand. Maybe a good followup?

Reply:
![image](https://user-images.githubusercontent.com/625469/226800448-5871a976-e52c-43ee-a4a3-3318ca8bd635.png)
![image](https://user-images.githubusercontent.com/625469/226800484-ddf3f16f-a511-436f-9cb4-9207c974f86e.png)

going with proper UI library will be nice, but this is oo scope for this change

Reply:
i refactored this code, that we should no longer need it